mlir/lib/Dialect/Rock/Transforms/ConvToGemm.cpp

```diff
@@ -321,16 +321,6 @@ struct ConvertingCopyKernelRewritePattern final
 
 /// Layout normalization.
 
-/// In most cases, we can treat the CYX or YXC parts of the filter dimension
-/// as a single unit for the purposes of vectorization, coordinate computations,
-/// and so on. However, if the filter layout is created by transposing the
-/// underlying memory in order to match another tensor, we cannot do this
-/// and so must flag when we have performed such a relayout.
-///
-/// This is a nominally temporary hack that should be removed once we can more
-/// precisely handle Unfold{} by static analysis instead of by heuristic.
-/// As such, it'll probably presist for quite some time.
-constexpr llvm::StringLiteral dontUnfoldAttrName("has_relayout_do_not_unfold");
```

Comment:
Re: fusion refactor, 
ah ha! we had this defined and explained but buried here... 
anyways, doesn't matter as it goes away!